script/Arcueid/Day 8/08_00_ARC08_5_1.txt

```diff
@@ -82,7 +82,8 @@ With that, the conversation comes to an end.
 [sha:c43a9f56b4ac269b5e257deb07b4d6635fd59904]{
 -- Page 270, Offset 20343.
 -- 俺は翡翠に頭を下げて、ロビーの階段を昇っていく。
-I bow my head to Hisui, and start to climb the lobby stairs.
+// or just remove the whole "to her" part
+I bow my head to her, and start to climb the lobby stairs.
```

Comment:
I think you need to keep the fact that it's towards Hisui. Otherwise it reads like he's hanging his head in shame.